src/components/layout/right-menu.vue

```diff
@@ -21,7 +21,16 @@
 </template>
 
 <script>
-export default {};
+import { mapMutations, mapState } from "vuex";
+
+export default {
+  computed: {
+    ...mapState("RightMenu", ["showAside"]),
+  },
+  methods: {
+    ...mapMutations("RightMenu", { toggleShowAside: "toggleShowRightAside" }),
```

Comment:
Usuń nieużywany kod. To są nieużywane metody w tym komponencie.

Reply:
Błąd jak wyżej